nomad/grapl-core.nomad

```diff
@@ -578,6 +630,71 @@ job "grapl-core" {
     }
   }
 
+  group "analyzer-dispatcher" {
+
+    task "analyzer-dispatcher" {
+      driver = "docker"
+
+      config {
+        image        = "${var.container_registry}/grapl/analyzer-dispatcher:${var.analyzer_dispatcher_tag}"
+        network_mode = "grapl-network"
+      }
+
+      env {
+        # AWS vars
+        AWS_REGION                  = var.aws_region
+        GRAPL_AWS_ACCESS_KEY_ID     = var.aws_access_key_id
+        GRAPL_AWS_ACCESS_KEY_SECRET = var.aws_access_key_secret
+        GRAPL_AWS_ENDPOINT          = var.aws_endpoint
+        # rust vars
+        RUST_LOG       = var.rust_log
+        RUST_BACKTRACE = 1
+        # service vars
+        GRAPL_ANALYZERS_BUCKET = var.analyzer_bucket
+        DEST_BUCKET_NAME       = var.analyzer_dispatched_bucket
+        DEAD_LETTER_QUEUE_URL  = var.analyzer_dispatcher_queue
+        SOURCE_QUEUE_URL       = var.analyzer_dispatcher_dead_letter_queue
+      }
+    }
+
+  }
+
+  group "analyzer-executor" {
+    task "analyzer-executor" {
+      driver = "docker"
+
+      config {
+        image        = "${var.container_registry}/grapl/analyzer-executor:${var.analyzer_executor_tag}"
+        network_mode = "grapl-network"
+      }
+
+      env {
+        # AWS vars
+        AWS_REGION                  = var.aws_region
+        GRAPL_AWS_ACCESS_KEY_ID     = var.aws_access_key_id
+        GRAPL_AWS_ACCESS_KEY_SECRET = var.aws_access_key_secret
+        GRAPL_AWS_ENDPOINT          = var.aws_endpoint
+        # python vars
+        GRAPL_LOG_LEVEL = "INFO"
+        # presumably not needed but will confirm after
+        RUST_LOG = "INFO"
```

Comment:
a) It's not needed, a-e is pure python for now.
b) i also hate that we have two disparate log level variabels and I want to fix it in the future - but that's a problem for Future Grapl